docs/release-notes/changelog.rst

```diff
@@ -75,6 +75,17 @@ Starting with Snapcraft 8.7.0, information about releases can be found in the
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+7.5.9 (2025-Sep-05)
+-------------------
+
+Core
+====
+
+* Fix a bug where Snapcraft would fail to re-use an existing build environment to pack
+a snap.
```

Comment:
Heads up, this should have been:

```suggestion
* Fix a bug where Snapcraft would fail to re-use an existing build environment to pack
  a snap.
```

I'll correct it when we pick up the changelog in the `docs/7` branch.